app/src/main/java/com/kickstarter/libs/FirebaseAnalyticsClient.kt

```diff
@@ -34,9 +34,7 @@ open class FirebaseAnalyticsClient(
 
     override fun sendUserId(user: User) {
         firebaseAnalytics?.let {
-            if (isEnabled()) {
-                firebaseAnalytics.setUserId(user.id().toString())
-            }
+            firebaseAnalytics.setUserProperty("ks_user_id", user.id().toString())
```

Comment:
Can still be useful in the future so letting it here but let me know if there is any concern, using unique ID's is not possible currently for Firebase Remote Config, but for groups such as admin users or super backers can potentially be useful in the future